code/game/gamemodes/cult/cult_items.dm

```diff
@@ -40,7 +40,7 @@
 	flags_inv = HIDEFACE
 	flags = HEADCOVERSEYES
 	body_parts_covered = HEAD|EYES
-	armor = list(melee = 30, bullet = 10, laser = 5,energy = 5, bomb = 0, bio = 0, rad = 0)
+	armor = list(melee = 50, bullet = 30, laser = 50,energy = 20, bomb = 25, bio = 10, rad = 0)

```

Comment:
It is cult armor, aka, the robes are also armored, so, makes no sense for the hood to be armored as well.